assets/prototype/domain/eventEditor/tickets/TicketsList/filterBar/controls/TicketsSortedByControl.tsx

```diff
@@ -1,51 +1,59 @@
 /**
  * External imports
  */
-import * as React from 'react';
+import React from 'react';
 import { SelectControl } from '@wordpress/components';
-import { __ } from '@wordpress/i18n';
+import { __ } from '@wordpress/i18n'; // @todo: replace with '@eventespresso/i18n'
 
 /**
  * Internal imports
  */
-import { SortTickets } from '../../../../data/ticket/types';
+import { SortTicketsBy } from '../../../../data/ticket/types';
+
+interface TicketsSortedByControlProps {
+	sortTicketsBy: SortTicketsBy;
+	setSortTicketsBy: (sortTicketsBy: SortTicketsBy) => void;
+}
 
 /**
  * filter for controlling the sorting of a list of Tickets
  *
- * @param {Function} setTicketsSortedBy
- * @param {string} ticketsSortedBy
+ * @param {Function} setSortTicketsBy
+ * @param {string} sortTicketsBy
  * @return {Object} rendered control
  */
-const TicketsSortedByControl = ({ setTicketsSortedBy, ticketsSortedBy }) => {
+const TicketsSortedByControl: React.FC<TicketsSortedByControlProps> = ({
+	setSortTicketsBy,
+	sortTicketsBy,
+}): JSX.Element => {
 	return React.useMemo(() => {
 		return (
 			<SelectControl
 				label={__('sort', 'event_espresso')}
 				className='ee-ticket-list-filter-bar-order-select'
-				value={ticketsSortedBy}
+				value={sortTicketsBy}
 				options={[
 					{
-						value: SortTickets.chronologically,
+						value: SortTicketsBy.chronologically,
 						label: __('chronologically', 'event_espresso'),
 					},
 					{
-						value: SortTickets.byName,
+						value: SortTicketsBy.byName,
```

Comment:
Not sure if how this ended up reads nice.
SortTickets`by by`Name
we might consider updating to:
`SortTicketsBy.name`

Reply:
also not sure about:
`SortTicketsBy.chronologically`

Reply:
I wanted to make it `SortTicketsBy.name` but it would make the PR huge

Reply:
`SortTicketsBy.chronologically` would be `SortTicketsBy.chronology`

Reply:
> but it would make the PR huge

I am ok with the current state then

Reply:
how about `SortTickets.byName`, `SortTickets.chronologically`, etc, etc ?

Reply:
I have no preference for that but I thought to make it consistent with `DatesSortedBy`. IMHO adding that `By` at the end makes it clear what the variable/type means.

Reply:
I'll create another issue for improving these names